setup.py

```diff
@@ -48,22 +48,30 @@
         'Topic :: Software Development :: Libraries :: Python Modules',
     ],
     install_requires = [
-        'Django==1.8.13',
-        'django-autocomplete-light==2.3.3',
-        'django-extensions>=1.5.5',
-        'django-nose>=1.4',
+        'Django>=1.8.2',
```

Comment:
> Instead of pinning it to a specific Django patch level, would you accept Django>=1.8,<1.9 so we can get the most recent patchlevel of Django 1.8?

Sure

Btw maybe you could do sth like this:
in base requirements in setup.py require Django>=1.8 and in dnsaas requirements require Django<1.9? We have to install dnsaas dependencies either way, and if you don't install it, you will end up with the newest django. Will it work? :)